src/components/menu/scrollable-block/scrollable-block.component.js

```diff
@@ -46,25 +57,16 @@ const ScrollableBlock = ({
         as="ul"
         role="list"
       >
-        {React.Children.map(children, (child, index) => {
-          let isFocused = false;
-          const blockItemFocused = focusIndex >= blockIndex;
-
-          if (blockItemFocused) {
-            isFocused = focusIndex - blockIndex === index;
-          }
-
-          return (
-            <SubmenuContext.Provider
-              value={{
-                isFocused,
-                handleKeyDown,
-              }}
-            >
-              {child}
-            </SubmenuContext.Provider>
-          );
-        })}
+        <SubmenuContext.Provider
```

Comment:
do we really have to wrap the children in another context? We're already inside a `SubmenuContext` with exactly these values (since they come from the `useContext` call above), so surely the children will be able to access the same values via `useContext`?

Reply:
Nope you are right, I only kept it as it was implemented like that already and was trying to limit the changes (too late really 🤣). Removed and functionality doesn't seem to be broken